### DISCUSSION TITLE
Ruleset for beatmap info overlay and user ranks defaulting to osu! game mode in real-time multiplayer

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16555] [Q&A] [answered]
Version 2022.108.0
When clicking on map links, it defaults to displaying the osu! difficulties (if there are any) EVEN IF the correct difficulty is selected. As seen in https://youtu.be/DDVeIYY3pYw?t=1275 the beatmap metadata is the correct difficulty, but the display shows the osu! difficulties anyway.
Additionally, user ranks are displayed for the osu! game mode regardless of what primary game mode is being played or the default game mode of that user's profile. (Source: https://youtu.be/DDVeIYY3pYw?t=4965)
![image](https://user-images.githubusercontent.com/74262407/150625688-84d8f492-b734-42ce-b3c3-aed60ba7b36a.png)

**Comment by bdach** [comment, marked as answer]
tracking as https://github.com/ppy/osu/issues/16566 and ~~https://github.com/ppy/osu/issues/16565~~ https://github.com/ppy/osu/issues/3606

going to leave these on the backburner for now as they don't seem to be critical.

**Reply by zachmanthethird** [reply]
Whoops, I don't think I explained myself enough on the user portion (though it may still have the same answer).
Stats are presented based on that user's chosen default game mode on the profile overlay. In contrast, the multiplayer lobby screen shows players' ranks exclusively in the osu! game mode regardless of current map mode, ruleset selected, and profile game mode.
Whether these systems should be segregated, I don't know. However, I agree that this isn't an immediate problem in any way :+1:

**Reply by bdach** [reply]
oh, right. there's actually a todo for that too:

https://github.com/ppy/osu/blob/70e818ff8f5f5e54aca55752b37d71e46115de73/osu.Game/Screens/OnlinePlay/Multiplayer/Participants/ParticipantPanel.cs#L187-L188

**Reply by zachmanthethird** [reply]
For searchability purposes, this is being tracked via #16730. :+1:
